src/SummaryCell.tsx

```diff
@@ -37,7 +37,7 @@ function SummaryCell<R, SR>({
     typeof summaryCellClass === 'function' ? summaryCellClass(row) : summaryCellClass
   );
 
-  function onClick() {
+  function onPointerDown() {
     selectCell({ rowIdx, idx: column.idx });
```

Comment:
I wonder if we should have unified event handlers at the DataGrid level instead of having the same implementations in various components.
Something like
```ts
function onCellEvent(event: any event, pointerArgs: which row/cell) {
  // if pointer down...
  // if click...
}
```

Reply:
I was thinking the when I added https://github.com/adazzle/react-data-grid/pull/3774#discussion_r2067467182. I will check